lib/beaker/hypervisor/vmpooler.rb

```diff
@@ -246,5 +273,82 @@ def cleanup
       @logger.notify "Spent %.2f seconds cleaning up" % (Time.now - start)
     end
 
+    def add_disk(hostname, disk_size)
+      @logger.notify "Requesting an additional disk of size #{disk_size}GB for #{hostname}"
```

Comment:
@eputnam would you put in a sentence to this effect in the doc for adding disks above? Just reading the docs, I'm not seeing what the numbers mean above. This line of code clarifies that, but we have a problem in beaker where people typically have to look at the code to figure out what's going on because our docs aren't clear enough.